pkg/policy/policy.go

```diff
@@ -259,13 +267,13 @@ func (p *Policies) Eval(ctx context.Context, resource client.Object) ([]v1alpha1
 			return nil, fmt.Errorf("failed evaluating Rego deny rule: %s: %w", denyQuery, err)
 		}
 
-		denyResults, hasDenyResults := hasBindings(deny, "res")
-		if hasDenyResults {
-			denyChecks, err := valuesToChecks(md, denyResults)
+		denyValues, hasDenyValues := hasBindings(deny, "res")
```

Comment:
maybe make "res" a constant with description comment just to make it purpose clear for new readers

Reply:
Done